crates/hyperswitch_connectors/src/connectors/nuvei/transformers.rs

```diff
@@ -1404,7 +1404,7 @@ where
             card_holder_name: data.card_holder_name,
             expiration_month: Some(data.card_exp_month),
             expiration_year: Some(data.card_exp_year),
-            ..Default::default() // CVV should be disabled by nuvei fo
+            ..Default::default() // CVV should be disabled by nuvei for
```

Comment:
```suggestion
            ..Default::default() // CVV should be disabled for nuvei
```

correct me if wrong, @Nithin1506200

Reply:
also, you do not need to merge from main when this branch goes out of date with the base branch (main).